### ISSUE TITLE
Weird duplicating of window buttons

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#3198]
Problem solved with a restart. Just FYI. Probably due to sync issue - i have a new phone and had some issues with syncing

![image](https://github.com/AndBible/and-bible/assets/13920678/47e0b8a4-c57a-44e9-ac07-ef8745020383)

![image](https://github.com/AndBible/and-bible/assets/13920678/35135fd6-48a2-462d-878d-d0a242129795)

**Comment by tuomas2** [comment]
Did you happen to get a logcat of this?

**Comment by agrogers** [comment]
I don't think so. Next time I will remember to do it. I have never seen it again... it was just for your info so that if you saw it from another person you could raise its importance.
